cmd/machine-config-operator/bootstrap.go

```diff
@@ -24,28 +24,34 @@ var (
 	}
 
 	bootstrapOpts struct {
-		etcdCAFile          string
-		rootCAFile          string
-		kubeCAFile          string
-		pullSecretFile      string
-		configFile          string
-		oscontentImage      string
-		infraConfigFile     string
-		networkConfigFile   string
-		imagesConfigMapFile string
-		mccImage            string
-		mcsImage            string
-		mcdImage            string
-		etcdImage           string
-		setupEtcdEnvImage   string
-		infraImage          string
-		destinationDir      string
+		etcdCAFile                string
+		etcdMetricsCAFile         string
+		etcdMetricsServerCertFile string
+		etcdMetricsServerKeyFile  string
+		rootCAFile                string
+		kubeCAFile                string
+		pullSecretFile            string
+		configFile                string
+		oscontentImage            string
+		infraConfigFile           string
+		networkConfigFile         string
+		imagesConfigMapFile       string
+		mccImage                  string
+		mcsImage                  string
+		mcdImage                  string
+		etcdImage                 string
+		setupEtcdEnvImage         string
+		infraImage                string
+		destinationDir            string
 	}
 )
 
 func init() {
 	rootCmd.AddCommand(bootstrapCmd)
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdCAFile, "etcd-ca", "/etc/ssl/etcd/ca.crt", "path to etcd CA certificate")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdMetricsCAFile, "etcd-metrics-ca", "", "path to etcd metrics CA certificate")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdMetricsServerCertFile, "etcd-metrics-server-crt", "", "path to etcd metrics server certificate")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.etcdMetricsServerKeyFile, "etcd-metrics-server-key", "", "path to etcd metrics server key")
```

Comment:
AIUI, these flags should provide a dummy/default value for this PR to _not_ depend on the installer. After this goes in, you can land the installer PR and then create another PR here to remove the dummy/default value here to always honor what bootkube gives you.

(this is not changing or adding images from the release payload so it should be an easier dance)

/cc @cgwalters

Reply:
ok sounds good that was my thought as well.